en/MDN_Plus_Terms

```diff
@@ -0,0 +1,71 @@
+# MDN Plus  Terms of Service
+
+Version 1.0, effective March 9, 2022
+{: datetime="2022-03-09" }
+
+MDN Plus is a subscription service providing premium functionality, including MDN offline, bookmarking, themes, and notifications. (the “Service”). Please read these Terms of Service carefully because they explain important information about your use of the Service.
+
+## You Must Be Eligible to Use the Service
+
+You must be in a country where the Service is available. Currently, the Service is available in the United States, Canada, the United Kingdom, Germany, France, Italy, Spain, Belgium, Austria, Switzerland, the Netherlands, Ireland, Malaysia, New Zealand, and Singapore. A Firefox Account is required to use the Service.
+To create a Firefox Account, you will also need to agree to the [Terms of Service](https://www.mozilla.org/about/legal/terms/services/) and [Privacy Notice](https://www.mozilla.org/privacy/firefox/) for your Firefox Account.
+
+## Your Privacy
+
+__Your Privacy.__ The [MDN Plus Privacy Notice]() explains what information is sent when you use the Service and how we handle and share that information.
```

Comment:
I assume a MDN Plus PN URL will be inserted soon.

Reply:
yes. I still need to ask the website team to generate the URLs